### ISSUE TITLE
Add linter to NuRadioMC

### ISSUE THREAD

**Issue by anelles** [nu-radio/NuRadioMC#258]
Since we want to merge NuRadioReco over, we should also move the linter functionality over. Everyone needs to help though cleaning up the code. 
@christophwelling can you start in a PR and then we all help?
Or who volunteered to do this? I forgot ... sorry :/

**Comment by christophwelling** [comment]
The Linter is already part of the new tests, but it only fails the test if there are serious issues.
We should do the code cleanup last, otherwise this will produce to many merge conflicts for the other PRs.

**Comment by cg-laser** [comment]
agreed, and I'm also inclined to not be too strict on syntax. I think failing only on critical issues might be a good way. The linter still tells us other warnings. In any case, I would leave it like that for now.
